.gitignore

```
@@ -36,7 +36,7 @@ compile_commands.json
 platform/windows/godot_res.res
 
 # Ninja build files
-build.ninja
+*.ninja
 .ninja
```

`*` also covers 0 chars, so the second one is redundant:
```suggestion
*.ninja
```

---

Second one is directory so I decided to leave just in case

---

It seems to be properly ignored with my suggested change, but yeah we could keep it explicit, then I'd go one step further and make it `.ninja/` so it's clear it's a directory.